packages/forklift-console-plugin/build-info.json.template

```diff
@@ -0,0 +1,12 @@
+{
+  "package": {
```

Comment:
a. can you follow the `@openshift/dynamic-plugin-sdk` example and use the same json struct they use ?
b. can you update our rollup plugin, and add the missing `commit` and `branch` fields [2] ?

[1] https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/common/rollup/rollup-configs.js#L22
[2] https://github.com/kubev2v/forklift-console-plugin/blob/main/config/rollup.base.js#L46

p.s.
the reason I removed this fields was because `execSync` is not a nice command, but we can use it if we need. you may see some linter warnings about using it, if you do, you can ignore them.

Reply:
> a. can you follow the `@openshift/dynamic-plugin-sdk` example and use the same json struct they use ? 
> 
> [1] https://github.com/openshift/dynamic-plugin-sdk/blob/main/packages/common/rollup/rollup-configs.js#L22 
>

Their structure is flat and ugly.  Plus their date/time is stuck to whatever TZ the builder is using.  The dateTime used here is always ISO 8601 format.  Much nicer.

> b. can you update our rollup plugin, and add the missing `commit` and `branch` fields [2] ?
>
> [2] https://github.com/kubev2v/forklift-console-plugin/blob/main/config/rollup.base.js#L46
> 

I think you covered this in #457

> p.s. the reason I removed this fields was because `execSync` is not a nice command, but we can use it if we need. you may see some linter warnings about using it, if you do, you can ignore them.

I used `isomorphic-git` which is pure JS and therefore much safer to use in the build.

Reply:
Dropped `isomorphic-git` so people can read the git commands they're used to.